syft/pkg/cataloger/golang/test-fixtures/archs/Makefile

```diff
@@ -0,0 +1,8 @@
+GO_1_17 = /Users/jonas/.gvm/gos/go1.17.8/bin/go
```

Comment:
this is a hardcoded value to your user. If the problem is needing to build with specific versions of go, we should probably containerize these steps.